tendermint/src/lib.rs

```diff
@@ -35,6 +35,7 @@ pub mod chain;
 pub mod channel;
 pub mod consensus;
 pub mod crypto;
+pub mod deserializers;
```

Comment:
Let's add it to the `serializers` instead, to help discovery.

Reply:
Done in 568f843becc80bce00cf12fa1bf58eebfcba273d